### DISCUSSION TITLE
Welcome to chainlink-mix Discussions!

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [smartcontractkit/chainlink-mix#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Hello!

just a little typo found on your README.md guide

![image](https://user-images.githubusercontent.com/89277831/145260659-d80fb225-a857-44e6-a79f-e3be0afeb155.png)

the folder is **cd chainlink** instead cd chainlink-mix

thank y!

**Reply by PatrickAlphaC** [reply]
Oh thank you! Could you make a PR to fix?

**Comment by ritesh798** [comment]
I'm facing an issue while running "brownie bake chainlink-mix" command please hel
![browniechainlink](https://user-images.githubusercontent.com/69509946/164682340-510d8b63-f370-4326-b1bc-274a79e4260c.png)
p

**Reply by PatrickAlphaC** [reply]
Thanks! This is likely an issue with something with your setup. Can you please ask on StackOverflow or stack exchange ETH?
